filters/include/pcl/filters/impl/local_maximum.hpp

```diff
@@ -110,28 +110,28 @@ pcl::LocalMaximum<PointT>::applyFilterIndices (std::vector<int> &indices)
   // Find all points within xy radius (i.e., a vertical cylinder) of the query
   // point, removing those that are locally maximal (i.e., highest z within the
   // cylinder)
-  for (int iii = 0; iii < static_cast<int> (indices_->size ()); ++iii)
+  for (const auto& iii : (*indices_))
   {
-    if (!isFinite ((*input_)[(*indices_)[iii]]))
+    if (!isFinite ((*input_)[iii]))
```

Comment:
`isXYZFinite((*input_)[iii])`  (explicit instead of the implicit version)

And same in other places. This can go in a separate PR, but since these lines are already being touched, I'm flagging them here

Reply:
Yes, that is best left for another PR, since it is not really related